### DISCUSSION TITLE
How to customize, in a sane way, the Divider inside the Select?

### DISCUSSION THREAD

**Discussion by hensansi** [mantinedev/mantine#3360] [Q&A] [answered]
Currently we have the Divider with a very light color prop and I noticed that it was too light for the Select, so I tried to customize it through the defaultProps but I wasn't able to.

Do you know if it is possible to do so without the "!important" or the ":not("some-class-that-might-change)"?
Is it possible to customize the behaviour of the component inside the component through the themes?

https://codesandbox.io/s/mantine-select-divider-jfgish?file=/src/App.tsx

**Comment by rtivital** [comment, marked as answer]
What is wrong with `!important`? It is made exactly for this use case.

**Reply by hensansi** [reply]
With Mantine we typically are able to customize the props of the components through props, so I was wondering if there was a way to do so. 

There is nothing wrong with "!important" besides the fact that it tends to be a last resort...

I just remembered one way, even though it won't be done at the app theme level, just the component one, I can wrap the Select component with a MantineProvider feeding the same theme but without the Divider customization. Are there any consequences in doing so?

```
<MantineProvider inherit theme={{ components: { ...APP_THEME.components, Divider: undefined } }}>
  <Select />
</MantineProvider>
```

**Reply by rtivital** [reply]
I advise just to use !important, it is the right tool for the job
